feature-libs/organization/administration/core/store/effects/org-unit.effect.ts

```diff
@@ -464,33 +464,37 @@ export class OrgUnitEffects {
     )
   );
 
-  // @Effect()
   // loadAddress$: Observable<
   //   | OrgUnitActions.LoadAddressSuccess
   //   | OrgUnitActions.LoadAddressesSuccess
   //   | OrgUnitActions.LoadAddressesFail
-  // > = this.actions$.pipe(
-  //   ofType(OrgUnitActions.LOAD_ADDRESSES),
-  //   map((action: OrgUnitActions.LoadAddresses) => action.payload),
-  //   switchMap(({ userId, orgUnitId }) => {
-  //     return this.orgUnitConnector.getAddresses(userId, orgUnitId).pipe(
-  //       switchMap((addresses: EntitiesModel<B2BAddress>) => {
-  //         const { values, page } = StateUtils.normalizeListPage(addresses, 'id');
-  //         return [
-  //           new OrgUnitActions.LoadAddressSuccess(values),
-  //           new OrgUnitActions.LoadAddressesSuccess({ page, orgUnitId }),
-  //         ];
-  //       }),
-  //       catchError(error =>
-  //         of(
-  //           new OrgUnitActions.LoadAddressesFail({
-  //             orgUnitId,
-  //             error: normalizeHttpError(error),
-  //           })
+  // > = createEffect(() =>
```

Comment:
The dead code should probably just be removed.

Reply:
removed